doc/sphinx/source/animations.rst

```diff
@@ -79,11 +80,7 @@ Let's have a detailed look at the ``idle`` animation:
    *Optional*. The frames per second for this animation if you want to deviate from the default fps. This will control the playback speed of the animation. Do not specify this value if you have only 1 animation frame.
 
 **sound_effect**
-   *Optional*. Our example will look for the sound files ``bar_00.ogg`` through ``bar_99.ogg`` in the directory ``data/sound/foo`` and play them in sequence. The priority is optional with the default being ``1``, and its range is:
-
-   * **0-127:** Probability between ``0.0`` and ``1.0``, only one instance can be playing at any time
-   * **128-254:** Probability between ``0.0`` and ``1.0``, many instances can be playing at any time
-   * **255:** Always play
+   *Optional*. Our example will look for the sound files ``bar_00.ogg`` through ``bar_99.ogg`` in the directory ``data/sound/foo`` and play one of them at random. The priority ranges from ``0.01`` to ``100`,` and it is the percent chance of the sound being played. If ``allow_multiple = true``, the same sound can be played by multiple map objects at the same time.
```

Comment:
`Optional` → `Mandatory`

Reply:
No, the sound effect itself is optional. What is mandatory is that if you specify a sound effect, then you have to specify the sound file and the priority.